### ISSUE TITLE
Перестал работать плагин ! 

### ISSUE THREAD

**Issue by Laktor90** [LinFor/jellyfin-plugin-kinopoisk#32]
Обновился до последней версии через офф репозиторий на jellyfin. После обновления плагин не подтягивает описание с КиноПоиска. Стояла какая то старая версия, все работало. Ограничение на api снято.

**Comment by jahlib** [comment]
попробуй этот репозиторий
https://github.com/skrashevich/jellyfin-plugin-kinopoisk
там фикс от уважаемого @ThinkAndRun

**Comment by RomanAverin** [comment]
Same issue

**Comment by Lekos555** [comment]
Да, на 10.9.7 не работает, загрузка только основного изображения. Свой API токен прописан.

**Comment by Lekos555** [comment]
> попробуй этот репозиторий https://github.com/skrashevich/jellyfin-plugin-kinopoisk там фикс от уважаемого @ThinkAndRun

Не работает так же.

**Comment by mirlex1986** [comment]
с форком от уважаемого @VD42 все работает. Плюс если починить плагин TMDB, то очень много инфы в метаданных становится

**Comment by lilPrince** [comment]
> с форком от уважаемого @VD42 все работает. Плюс если починить плагин TMDB, то очень много инфы в метаданных становится

да, спасибо, вроде заработало! а как починить тмдб?

**Comment by Sivel0302** [comment]
> с форком от уважаемого @VD42 все работает.

Подтверждаю - всё заработало)) Спасибо, чувак!)

**Comment by jahlib** [comment]
> починить плагин TMDB

не подскажите как починить?

**Comment by mirlex1986** [comment]
> > починить плагин TMDB
> 
> 
> 
> не подскажите как починить?

У меня работают с dns 64.6.64.6
